docs/requirement-sets/outlook/requirement-set-1.5/outlook-requirement-set-1.5.md

```diff
@@ -32,6 +32,12 @@ Requirement set 1.5 includes all of the features of [requirement set 1.4](../req
 - Modified [Office.context.mailbox.item.addFileAttachmentAsync](office.context.mailbox.item.md#methods): A new value in the `options` dictionary called `isInline`, used to specify that an image is used inline in the message body.
 - Modified [Office.context.mailbox.item.displayReplyAllForm](office.context.mailbox.item.md#methods): A new value in the `formData.attachments` dictionary called `isInline`, used to specify that an image is used inline in the message body.
 - Modified [Office.context.mailbox.item.displayReplyForm](office.context.mailbox.item.md#methods): A new value in the `formData.attachments` dictionary called `isInline`, used to specify that an image is used inline in the message body.
+- Added [Office.ContextInformation](/javascript/api/office/office.contextinformation?view=outlook-js-1.5&preserve-view=true): Provides information about the environment in which the add-in is running.
+- Added [Office.context.diagnostics](office.context.md#properties): Gets information about the environment in which the add-in is running, including host, platform, and version information.
+- Added [Office.context.host](office.context.md#properties): Gets the Office application that is hosting the add-in.
+- Added [Office.context.platform](office.context.md#properties): Gets the platform on which the add-in is running.
```

Comment:
For these, can you link directly to the property's anchor on the page instead of the property table?